enhancements/console/dynamic-plugins.md

```diff
@@ -87,10 +87,12 @@ to dynamically load code from another application while sharing dependencies.
 It also allows console to share its components with plugins built and bundled
 separately.
 
-Plugins will need to be built with Webpack 5 to use module federation.
+Plugins will need to be built with Webpack 5+ which includes native support for
+module federation.
 
-We have a pull request with a [working prototype](https://github.com/openshift/console/pull/6101)
```

Comment:
I've removed this section since openshift/console#6101 is already merged and listed below in "Implementation History" section. If you think we should keep it, let me know and I'll restore it.